src/collections/_documentation/data-management/advanced-datascrubbing.md

```diff
@@ -0,0 +1,425 @@
+
+**Advanced Data Scrubbing is currently only available if your organization is an Early Adopter.** To become one, go to your organization settings and enable the "Early Adopter" option to get access to new features before they're fully released. This option can be just as easily disabled as it can be enabled.
+
+You will then get access to one new option in both organization settings and the settings of each project called "Advanced datascrubber configuration". Its content is a piece of JSON-based configuration in the format described in the next section.
+
+Note that everything you configure there will have direct impact on all new events, just as all the other data privacy-related settings do. However, it is not possible to break or undo any other data privacy settings that you may have configured. In other words, it is only possible to accidentally remove too much data, not too little.
+
+If you have any questions related to this feature, feel free to contact us at `markus@sentry.io`.
+
+## A basic example
+
+Say you have an exception message which, unfortunately, contains IP addresses which are not supposed to be there. You'd write:
+
+```json
+{
+  "applications": {
+    "$string": ["@ip:replace"]
+  }
+}
+```
+
+It reads as "replace all IP addresses in all strings", or "apply `@ip:replace` to all `$string` fields".
+
+`@ip:replace` is called a rule, and `$string` is a [_selector_](#selectors).
+
+## Built-in rules
+
+The following rules exist by default:
+
+- `@ip:replace` and `@ip:hash` for replacing IP addresses.
+- `@imei:replace` and `@imei:hash` for replacing IMEIs
+- `@mac:replace`, `@mac:mask` and `@mac:hash` for matching MAC addresses
+- `@email:mask`, `@email:replace` and `@email:hash` for matching email addresses
+- `@creditcard:mask`, `@creditcard:replace` and `@creditcard:hash` for matching creditcard numbers
+- `@userpath:replace` and `@userpath:hash` for matching local paths (e.g. `C:/Users/foo/`)
+- `@password:remove` for removing passwords. In this case we're pattern matching against the field's key, whether it contains `password`, `credentials` or similar strings.
+- `@anything:remove`, `@anything:replace` and `@anything:hash` for removing, replacing or hashing any value. It is essentially equivalent to a wildcard-regex, but it will also match much more than strings.
+
+## Writing your own rules
+
+Rules generally consist of two parts:
+
+- *Rule types* describe what to match. See [_Rule Types_](#rule-types) for an exhaustive list.
+- *Rule redaction methods* describe what to do with the match. See [_Redaction Methods_](#redaction-methods) for a list.
+
+Each page comes with examples. Try those examples out by pasting them into the "PII config" column of [_Piinguin_](https://getsentry.github.io/piinguin) and clicking on fields to get suggestions.
+
+## Selectors
+
+You have the possibility to select a region of the event using JSON-path-like syntax. To delete a specific key in "Additional Data", you could for example use:
```

Comment:
Suggest "You can select a region of the event using JSON-path-like syntax. As an example, to delete a specific key in "Additional Data":